accounts/forms.py

```diff
@@ -327,3 +319,18 @@ def __init__(self, *args, **kwargs):
                 }
         super(DeleteUserForm, self).__init__(*args, **kwargs)
 
+
+class EmailSettingsForm(forms.Form):
+    enabled_stream_emails = forms.BooleanField(
```

Comment:
ok I see, then we should support both behaviours (default send and default not send). This is basically done in the logic code before sending an email, but it should be reflected in preferences definition somehow. Maybe a good option is that the email types that we have include the action. So we could rename the EmailType to EmailPreferenceType and then make the names like "UnmuteStreamEmails", "MuteSoundCommentsEmails". I think this makes sense.